tests/src/test/java/tachyon/client/FileInStreamIntegrationTest.java

```diff
@@ -56,6 +57,9 @@
   private static CreateFileOptions sWriteUnderStore;
 
   @Rule
+  public Timeout mGlobalTimeout = Timeout.seconds(60);
```

Comment:
Try it out with an hdfs profile, and see how long the tests take. I remember the hdfs profiles always take longer...